docs/remote_config.md

```diff
@@ -0,0 +1,28 @@
+# Remote Configuration (beta)
+
+This feature was introduced in Datadog Operator v1.7.0 and is currently in beta.
+
+## Overview
+
+Remote Configuration in the Datadog Operator allows you to enable features in a Kubernetes cluster from Datadog.
+
+You can create a policy on the Fleet Automation page to enable a feature for an eligible scope. The Datadog Operator then updates the Agents with the necessary configuration.
```

Comment:
Should we reference some public doc here? It wouldn't be obvious what policy or scope means for people new to RC.
E.g. one of these or both
https://docs.datadoghq.com/agent/remote_config/
https://www.datadoghq.com/blog/remote-configuration-for-datadog/

Reply:
We decided not to intentionally during this beta